### PR TITLE
DW-979-fix-calculator-switch-tabs

### PR DESCRIPTION
**Pull request by silsanchez8**
## Modifications:

* fix(plan-service): Show subscriber tab when is a montly-deliveries user
* Fix plan-service test.

image

### CODE REVIEW

**Comment by jcamposmk** [comment]
image

**Comment by cbernat** [comment]
👍

**Comment by silsanchez8** [comment]
test this please

File: src/services/plan-service.test.js [hunk 1]
```diff
@@ -341,7 +341,7 @@ describe('Doppler plan client', () => {
     var plans = await planService.getPlans(currentPlan, 'standard', 'subscribers', planList);
```
Comment (jcamposmk) [inline src/services/plan-service.test.js:341]:
```suggestion
const plans = await planService.getPlans(currentPlan, 'standard', 'subscribers', planList);
```

Comment (silsanchez8) [reply src/services/plan-service.test.js:341]: Ok,  @jcamposmk  good catch! this test file uses `var` in many places, when it declares `var plans = await planService.getPlans` . What do you think about change `var` for `const ` in all that places?

Comment (silsanchez8) [reply src/services/plan-service.test.js]: Changes applied @jcamposmk  😉

File: src/services/plan-service.ts [hunk 2]
```diff
@@ -170,10 +170,13 @@ const getPotentialUpgrades = (userPlan: Plan, planList: Plan[]): Plan[] => {
           ...getUpgradeSubscribersPlans(planList),
         ]
       : userPlan.type === 'monthly-deliveries'
-      ? getUpgradeMonthlyPlans(planList, {
-          minFee: userPlan.fee,
-          minEmailsByMonth: userPlan.emailsByMonth,
-        })
+      ? [
```
[the remark this answers is not in the document]
Comment (silsanchez8) [reply src/services/plan-service.ts:169]: It's a good suggestion, more readable and semantic. I will apply it. Thanks J! 👌

**Comment by silsanchez8** [comment]
test this please

**Comment by cbernat** [comment]
👍 🚀

**Pull request merged**